paddle/fluid/framework/executor.cc

```diff
@@ -484,5 +487,17 @@ void Executor::EnableMKLDNN(const ProgramDesc& program) {
 #endif
 }
 
+void Executor::EnableFusedOp(std::unique_ptr<ExecutorPrepareContext>& ctx) {
+#ifdef PADDLE_WITH_NGRAPH
+  VLOG(3) << "use_ngraph=True";
+  auto* fused_op = new FusedOperator(ctx->prog_, ctx->block_id_, ctx->ops_);
+  ctx->ops_.clear();
```

Comment:
I suggest you compare the ops in the ctx and the ops currently convertable to ngraph. If there is a op in ctx that ngraph cannot support, it should fail with a good error message.

Reply:
The updated code takes supported ops only in the fused op